packages/core/src/types/graphql.d.ts

```diff
@@ -0,0 +1,21 @@
+// This file is for align the types between frontend and backend API schema
+
+type ID = string;
+
+type Time = string;
+
+type Address = string;
```

Comment:
可以使用 `0x${string}` 更精確
```suggestion
type Address = `0x${string}`;
```

Reply:
已修正